core/iwasm/aot/aot_runtime.c

```diff
@@ -1802,7 +1802,7 @@ aot_invoke_native(WASMExecEnv *exec_env, uint32 func_idx, uint32 argc,
     bh_assert(func_idx < aot_module->import_func_count);
 
     import_func = aot_module->import_funcs + func_idx;
-    if (import_func->call_conv_wasm_c_api)
+    if (import_func->call_conv_wasm_c_api && c_api_func_import)
```

Comment:
are these NULL checks necessary?
as far as i know, when you have a function with call_conv_wasm_c_api, c_api_func_imports can't be NULL.

Reply:
> are these NULL checks necessary?

Good question, and frankly I don't know.

> when you have a function with call_conv_wasm_c_api, c_api_func_imports can't be NULL

How do we know this?

Reply:
Seems had better change to:
```C
if (import_func->call_conv_wasm_c_api)
    func_ptr = c_api_func_import ? c_api_func_import->func_ptr_linked : NULL;
```
Since for c api import, we shouldn't get func_ptr from `func_ptrs[func_idx]`